src/utils.ts

```diff
@@ -84,3 +85,7 @@ export const decodeBeaconValue = (encodedBeaconValue: string) => {
 
   return decodedBeaconValue;
 };
+
+export const getChainName = (chainId: string) => {
+  return CHAINS.find((c) => c.id === chainId)?.name ?? 'unknown';
```

Comment:
```suggestion
  return CHAINS.find((c) => c.id === chainId)?.alias ?? 'unknown';
```
I'd rather to utilize chainAlias instead of long chain name in the logs.